var/spack/repos/builtin/packages/clamav/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Clamav(AutotoolsPackage):
+    """Open source antivirus engine for detecting trojans,
+    viruses, malware & other malicious threats."""
+
+    homepage = "https://www.clamav.net/"
+    url      = "https://www.clamav.net/downloads/production/clamav-0.101.2.tar.gz"
+
+    version('0.101.2', sha256='0a12ebdf6ff7a74c0bde2bdc2b55cae33449e6dd953ec90824a9e01291277634')
+
+    depends_on('pkg-config', type='build')
+    depends_on('json-c')
+    depends_on('openssl')
+    depends_on('pcre')
+    depends_on('yara')
+    depends_on('zlib')
+    depends_on('bzip2')
+
+    def configure_args(self):
+        spec = self.spec
+        args = [
+            '--disable-dependency-tracking',
+            '--disable-silent-rules',
+            #--libdir=#{lib}
+            #--sysconfdir=#{etc}/clamav
+            '--disable-zlib-vcheck',
```

Comment:
Any idea why this is necessary? Based on the above flags, I have a feeling you took a Homebrew recipe and ported it to Spack.

Does anyone know what things like `--disable-dependency-tracking` and `--disable-silent-rules` actually do?? Every Homebrew package uses them, but most Spack packages don't. If we determine that they are useful, we could add them as default arguments in the `AutotoolsPackage` base class.

Reply:
yes, i just ported what Homebrew does and made sure it picks up all dependencies.

No idea what those `--disable-xyz` do.